letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -264,7 +264,18 @@ UNLIKELY_EOF
       # Report error. (Otherwise, be quiet.)
       echo "Had a problem while installing Python packages."
       if [ "$VERBOSE" != 1 ]; then
+        echo
+        echo "pip prints the following errors: "
+        echo "====================================================="
         echo "$PIP_OUT"
+        echo "====================================================="
+        echo
+        echo "Certbot has problem setting up the virtual environment."
```

Comment:
So maybe we should grep for the strings `Killed` and `allocate memory`, and print information suggesting that this *might* be the problem, and how to fix it, in those cases?